Formula/snappy.rb

```diff
@@ -18,10 +18,14 @@ class Snappy < Formula
   depends_on "pkg-config" => :build
 
   def install
-    system "cmake", ".", *std_cmake_args
+    args = *std_cmake_args + %w[
+      -DSNAPPY_BUILD_TESTS=OFF
+      -DSNAPPY_BUILD_BENCHMARKS=OFF
```

Comment:
As mentioned, this is one option that entirely disables the tests/benchmarks.
For now, trying a run with this.
This is the choice that MacPorts made: https://github.com/macports/macports-ports/blob/master/archivers/snappy/Portfile#L53-L56

Alternatively, we can go back to using Git Repo clone and try setting CMake options from the vendored libraries, i.e.
- [`-DBENCHMARK_ENABLE_INSTALL=OFF`](https://github.com/google/benchmark/blob/bf585a2789e30585b4e3ce6baf11ef2750b54677/CMakeLists.txt#L27)
- [`-DINSTALL_GTEST=OFF`](https://github.com/google/googletest/blob/18f8200e3079b0e54fa00cb7ac55d4c39dcf6da6/CMakeLists.txt#L26)

There could be another option outside these two.
The tests/benchmark feature is discussed: https://github.com/google/snappy#tests-and-benchmarks